src/clj/game/cards/assets.clj

```diff
@@ -1876,15 +1876,14 @@
                    (continue-ability
                      {:optional
                       {:prompt "Trash Rashida Jaheem to gain 3 [Credits] and draw 3 cards?"
-                       :msg "to trash itself"
+                       :msg "gain 3 [Credits] and draw 3 cards"
                        :yes-ability
                        {:async true
                         :effect (req (wait-for
                                        (trash state side card {:cause-card card})
                                        (wait-for
                                          (gain-credits state side 3)
                                          (wait-for (draw state side 3)
-                                                   (system-msg state side (str "uses " (:title card) " to gain 3 [Credits] and draw 3 cards"))
                                                    (effect-completed state side eid)))))}}}
```

Comment:
Given that you're removing the `system-msg`, you can simplify this to not use `wait-for`, something like `(wait-for (gain=-credit state side 3) (draw state side eid 3))`

Reply:
Do you mean like this?

```clojure
(wait-for
  (trash state side card {:cause-card card})
  (wait-for
    (gain-credits state side 3)
    (draw state side 3)
    (effect-completed state side eid)))
```

Reply:
Looks like you figured it out!